src/components/organisms/outputTabs.tsx

```diff
@@ -28,7 +28,7 @@ export const OutputTabs = () => {
 		<div className="flex h-full flex-col pb-3">
 			<div
 				className={
-					`static top-4 flex h-8 select-none items-center gap-1 uppercase xl:gap-2 2xl:gap-4 3xl:gap-5 ` +
+					`static -my-2 flex h-14 select-none items-center gap-1 uppercase xl:gap-2 2xl:gap-4 3xl:gap-5 ` +
```

Comment:
We should always avoid negative margins, is there any way we can make it without negative?